src/components/ClusterNodes/NodeList.tsx

```diff
@@ -68,6 +68,7 @@ export default function NodeList({ imageList, isSuperAdmin, namespaceList }: Clu
     const [nodeListOffset, setNodeListOffset] = useState(0)
     const [showTerminal, setTerminal] = useState<boolean>(false)
     const nodeList = filteredFlattenNodeList.map((node) => node['name'])
+    const clusterName: string = filteredFlattenNodeList[0] ? filteredFlattenNodeList[0]['clusterName'] : ''
```

Comment:
Change this to `filteredFlattenNodeList[0]?.['clusterName'] || ''`

Reply:
done